renovate.json

```diff
@@ -11,15 +11,40 @@
   },
   "masterIssue": true,
   "rebaseStalePrs": false,
-  "excludePackageNames": ["react", "react-dom"],
+  "ignoreDeps": ["react", "react-dom"],
   "rangeStrategy": "bump",
   "bumpVersion": null,
   "semanticCommitScope": null,
   "prHourlyLimit": 0,
   "packageRules": [
     {
       "groupName": "minor updates in packages",
-      "updateTypes": ["minor"]
+      "updateTypes": ["minor"],
+      "excludePackageNames": [
+        "// below is list of packages that use 0.X version range, any minor bump there can contain breaking changes, so we just ignore minor bumps for those packages and will need to bump them manually",
```

Comment:
I understand why we added a comment as a string but can we turn this into a `json5` file instead (by renaming it) [which Renovate supports](https://docs.renovatebot.com/configuration-options/) and in which we can have comments

Reply:
Good idea, testing it now in my fork